front/src/app/todo-list/components/active-task-group/active-task-group.component.ts

```diff
@@ -1,12 +1,63 @@
-import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component, Input, OnDestroy } from '@angular/core';
 import { TaskList } from '../../../shared/models/shared.models';
+import { Subject, takeUntil } from 'rxjs';
+import { ApiService } from '../../../shared/api-service/api.service';
 
 @Component({
   selector: 'app-active-task-group',
   templateUrl: './active-task-group.component.html',
   styleUrl: './active-task-group.component.scss',
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class ActiveTaskGroupComponent {
+export class ActiveTaskGroupComponent implements OnDestroy {
   @Input() taskList: TaskList | undefined;
+  @Input() activeGroupId: string = '';

```

Comment:
The naming is weird - why is it `taskList`, but `activeGroupId` - so is it `list` or `group`?

Same for component name and plenty other variables - please choose one word to use